lib/evaluate.js

```diff
@@ -122,6 +129,16 @@ export const evaluate = async ({
     //   point.floatField(`group_winning_${key}`, value)
     // }
   })
+
+  recordTelemetry('retrieval_stats', (point) => {
```

Comment:
can we make the distinction clearer like this for example:

```suggestion
  recordTelemetry('retrieval_stats_honest', (point) => {
```

Otherwise you don't know what the opposite of `_all` is, which are left out?